flux-image-reflector-controller.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: flux-image-reflector-controller
+  version: 0.28.0
+  epoch: 0
+  description: GitOps Toolkit controller that scans container registries
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/os
+    keyring:
+      - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/fluxcd/image-reflector-controller
+      tag: v${{package.version}}
+      expected-commit: 0a35c15fa055bcbf4625c5aa5e661516a64f1c61
+
+  - uses: go/build
+    with:
+      packages: main.go
```

Comment:
nit: if you switch this to the directory instead of the file we get slightly better embedded version metadata

Reply:
should I use "." in there?

Reply:
or "/" ?

Reply:
yep!

Reply:
done.